test/testdata/rewriter/def_delegator.rb

```diff
@@ -70,7 +70,7 @@ class IgnoredUsages
   not_def_delegator :thing, :foo # error: Method `not_def_delegator` does not exist
   def_delegator # error: Not enough arguments provided for method `Forwardable#def_delegator`
   def_delegator :thing
-  #             ^^^^^^ error: Not enough arguments provided for method `Forwardable#def_delegator`
+  #                   ^ error: Not enough arguments provided for method `Forwardable#def_delegator`
```

Comment:
Tests with sends without parentheses were the first thing that came to mind when I started scrolling through this, and I'm pleased to see that this change seems to do the right thing straightaway.  Would you mind adding some testcases to `arg_matching.rb.autocorrects.exp` so that no-parentheses calls have test coverage there, and not just in this random rewriter test?